sycl/doc/GettingStartedAlveo.md

```diff
@@ -0,0 +1,449 @@
+Check that the FPGA board is detected:
+```bash
+sudo /opt/xilinx/xrt/bin/xbutil flash scan -v
+XBFLASH -- Xilinx Card Flash Utility
+Card [0]
+	Card BDF:		0000:04:00.0
+	Card type:		u200
+	Flash type:		SPI
+	Shell running on FPGA:
+		xilinx_u200_GOLDEN_2,[SC=1.8]
+	Shell package installed in system:	(None)
+	Card name		AU200P64G
+	Card S/N: 		2130048BQ00H
+	Config mode: 		7
+	Fan presence:		P
+	Max power level:	75W
+	MAC address0:		00:0A:35:05:EF:0E
+	MAC address1:		00:0A:35:05:EF:0F
+	MAC address2:		FF:FF:FF:FF:FF:FF
+	MAC address3:		FF:FF:FF:FF:FF:FF
+```
+
+
+## Install SDAccel
+
+You need the framework to do the hardware synthesis, taking the SPIR
+intermediate representation (or HLS C++ or OpenCL or...) and
+generating some FPGA configuration bitstream.
+
+
+Download from
+https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/sdaccel-development-environment.html
+for example the web installer and use it with
+```bash
+chmod +x Xilinx_SDAccel_2019.1_0524_1430_Lin64.bin
+./Xilinx_SDAccel_2019.1_0524_1430_Lin64.bin
+```
+
+Select the `UltraScale+` platform for the U200 and install it to
+`/opt/xilinx` (because it is the default location used by XRT too).
+
+Ask for a 30 day trial license for example. It puts some information
+into `~/.Xilinx`.
+
+SDAccel comes with some old useless pieces of XRT to
+generate/instrospect the bitstream containers and still wants to use
+them... So we need to patch SDx to avoid using a wrong XRT by executing:
+```bash
+sudo ln -s $XILINX_XRT /opt/xilinx/SDx/2019.1/xrt/xrt-2.1.0-ubuntu19.04
+```
+
+
+## Install the shells for the FPGA board
+
+To execute some kernels on the FPGA board you need a deployment shell
+which is an FPGA configuration that pre-defines an architecture on the
+FPGA to execute some kernels in some reconfigurable area.
+
+To develop some kernels for your FPGA board and to run some
+simulations, you need a development shell that will content some
+internal description specific to the FPGA and its deployment shell so
+that the tools can generate the right bitstream for the kernels or the
+simulation details.
+
+Pick the latest deployment and development shells from
+https://www.xilinx.com/products/boards-and-kits/alveo/u200.html#gettingStarted
+for your board and for you OS. It might be an older version. For
+example it is possible to use a `2018.3` shell with SDAccel `2019.1`,
+and a version for Ubuntu `18.04` on a more recent version of Ubuntu or
+Debian.
+
+Install the shells:
+```bash
+dpkg -i xilinx-u200-xdma-201830.1_18.04.deb
+dpkg -i xilinx-u200-xdma-201830.1-dev_18.04.deb
+```
+from where they have been downloaded or adapt the paths.
+
+
+### Flash the board
+
+If you do not want to use a real board, skip this section.
+
+If you want to use a real board, follow the recipe "Generating the
+xbutil flash Command" from
+https://www.xilinx.com/html_docs/accelerator_cards/alveo_doc/ftt1547585535561.html
+about how to concretely generate the exact flashing command.
```

Comment:
maybe change concretely to correctly or perhaps just remove it, this might be me being picky though..